.github/workflows/verify.yml

```diff
@@ -1,35 +1,22 @@
 name: verify
 
 on:
-  pull_request:
-    paths-ignore:
-    - "**.md"
   push:
     branches:
     - '*'
     paths-ignore:
     - "**.md"
+  pull_request:
 
 jobs:
   verify:
     runs-on: ubuntu-latest
 
-    # update-codegen requires the workspace to be placed in GOPATH.
-    # Set up env var, working directory, and check out at the correct path in GOPATH.
-    env:
-      GOPATH: /home/runner/work/gardener-extension-shoot-flux/gardener-extension-acl/go
-    defaults:
-      run:
-        working-directory: go/src/github.com/stackitcloud/gardener-extension-acl
-
     steps:
     - uses: actions/checkout@v4
-      with:
-        path: go/src/github.com/stackitcloud/gardener-extension-acl
     - uses: actions/setup-go@v4
       with:
-        go-version-file: go/src/github.com/stackitcloud/gardener-extension-acl/go.mod
-        cache-dependency-path: go/src/github.com/stackitcloud/gardener-extension-acl/go.sum
+        go-version-file: go.mod
     - name: Configure git
       run: |
         git config --global user.email "ci@gardener.cloud"
```

Comment:
Is this intended to be a `@gardener.cloud` mail domain?

Reply:
This is just a dummy user that was once needed for gardener's `check-generate.sh` script.
I dropped this step from the workflow and replaced gardener's check-generate with dedicated make targets that are much simpler and don't write any commits.